selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/PullRequestPanel.java

```diff
@@ -64,6 +64,7 @@ public PullRequestPanel(
     static final String PULL_REQUEST_BTN = "gwt-debug-partButton-Pull Request";
     static final String BRANCH_NAME =
         "//div[text()='Branch name:']/following-sibling::div//span/label[text()='%s']";
+    static final String REFRESH_BRANCH_BUTTON = "gwt-debug-refreshContributionBranchButton";
```

Comment:
Is it button name or id? Coould you express it in the constant name to make it clear?